src/content/reference/management-api/appcatalogs.application.giantswarm.io.md

```diff
@@ -37,7 +37,8 @@ aliases:
 
 
 <h3 id="crd-example-v1alpha1">Example CR</h3>
-<pre class="crd-example-cr"><code class="language-yaml">
```

Comment:
Is this class needed for anything else?

Reply:
Looks like it was only in the html, no styling done with these classes (`crd-example-cr` and `language-yaml`)